client/my-sites/stats/videopress-stats-module/index.jsx

```diff
@@ -180,6 +183,9 @@ class VideoPressStatsModule extends Component {
 
 		return (
 			<div>
+				{ siteId && statType && query && (
```

Comment:
It is a little weird, but worth noting that the data component `QuerySiteStats` was [triggering in DownloadCsv](https://github.com/Automattic/wp-calypso/blob/6af7542d4c77371eca58f06bda9b478b66ef8955/client/my-sites/stats/stats-download-csv/index.jsx#L98-L100), so we need it when the Download CSV button is moved out.